app.py

```diff
@@ -0,0 +1,138 @@
+import base64
+import json
+import logging
+import os
+import sys
+import boto3
+
+from chalice import Chalice
+
+app_name = 'CMSFilterApp'
```

Comment:
I would consider putting Arc here instead of CMS since this function isn't CMS agnostic. something like FilterArcEventsApp